src/api-service/__app__/onefuzzlib/pools.py

```diff
@@ -923,7 +923,7 @@ def search_states(
 
     def update_nodes(self) -> None:
         # Be in at-least 'setup' before checking for the list of VMs
-        if self.state == self.init:
+        if self.state == ScalesetState.init:
```

Comment:
So this evaluated `False` 100% of the time, right? Why didn't mypy catch this?

Reply:
Not sure.  It's something to investigate offline.